esmvaltool/diag_scripts/ocean/diagnostic_maps.py

```diff
@@ -21,6 +21,19 @@
       time_average:
 
 
+Note that this recipe may not function on machines with no access to the
```

Comment:
I would recommend putting this information in doc/sphinx/source/recipes/recipe_oceans.rst instead of buried somewhere in the code.

Reply:
Done! I've written a section in that file. I would also recommending keeping this text in place though.